docs/installation.md

```diff
@@ -112,9 +113,17 @@ apt-get install build-essential redis-server libpng-dev git python3-minimal libv
 On Fedora/CentOS like:
 
 ```sh
-dnf install redis libpng-devel git libvhdi-tools lvm2 cifs-utils make automake gcc gcc-c++
+dnf install redis libpng-devel git lvm2 cifs-utils make automake gcc gcc-c++ nfs-utils
 ```
 
+:::tip
+You might need some additionnal packages depending on your use of XO.
+
+For instance, you may need _ntfs-3g_ if you are using NTFS disks.
+
+For development purposes, you may need _libvhdi-tools_ and _python3_ on Fedora/CentOS
```

Comment:
IMHO:
- do not document packages not necessary to build and run XO (e.g. packages used for tests)
- do not split out optional packages, let's keep this as simple as possible.

@fbeauchamp Why would you like them separate?